### ISSUE TITLE
Inter-item linking

### ISSUE THREAD

**Issue by pieterlexis** [isilher/chronagus#7]
It would be awesome to be able to link to (N)PC's and locations from log-entries (and the other way around maybe, e.g. having a list of log entries when looking at at a location or (N)PC) to quickly find info and have it logically linked together.

Bonus points if there is an on-hover card with a summary 😄

**Comment by isilher** [comment]
I fully agree: I was even planning to auto-link based on naming. The on-hover summary is a great addition!
